codec-framework/src/main/java/org/interledger/encoding/asn/framework/CodecContextFactory.java

```diff
@@ -50,50 +48,57 @@
  */
 public class CodecContextFactory {
 
-  public static final String OCTET_ENCODING_RULES = "OER";
-
   /**
-   * Get the {@link CodecContext} loaded with neccessary codecs and serializers for the given
-   * encoding rules.
+   * Create an instance of {@link CodecContext} which uses ASN.1 OER encoding.
    *
-   * @param encodingRules encoding rules to use for serialization
+   * @return A new instance of {@link CodecContext}.
+   */
+  public static CodecContext oer() {
```

Comment:
Thinking a bit more about this specific class, I don't see a need for a generic `register` method because this is a base factory. So, I think the only method we need in this class is this one:

```java
  /**
   * Constructs a {@link CodecContext} that is configured to read and write ASN.1 OER encodings.
   *
   * @return A {@link CodecContext}
   */
  public static CodecContext oer() {
    final AsnObjectCodecRegistry mappings = new AsnObjectCodecRegistry()
        .register(byte[].class, () -> new AsnOctetStringCodec(AsnSizeConstraint.UNCONSTRAINED))
        .register(Short.class, AsnUint8Codec::new)
        .register(Integer.class, AsnUint16Codec::new)
        .register(Long.class, AsnUint32Codec::new)
        .register(BigInteger.class, AsnUint64Codec::new)
        .register(String.class, () -> new AsnUtf8StringCodec(AsnSizeConstraint.UNCONSTRAINED));

    final AsnObjectSerializationContext serializers = new AsnObjectSerializationContext()
        .register(AsnCharStringBasedObjectCodec.class, new AsnCharStringOerSerializer())
        .register(AsnIA5StringCodec.class, new AsnCharStringOerSerializer())
        .register(AsnIA5StringBasedObjectCodec.class, new AsnCharStringOerSerializer())
        .register(AsnOctetStringCodec.class, new AsnOctetStringOerSerializer())
        .register(AsnOpenTypeCodec.class, new AsnOpenTypeOerSerializer())
        .register(AsnOctetStringBasedObjectCodec.class, new AsnOctetStringOerSerializer())
        .register(AsnSequenceCodec.class, new AsnSequenceOerSerializer())
        .register(AsnSequenceOfSequenceCodec.class, new AsnSequenceOfSequenceOerSerializer())
        .register(AsnUint8Codec.class, new AsnOctetStringOerSerializer())
        .register(AsnUint16Codec.class, new AsnOctetStringOerSerializer())
        .register(AsnUint32Codec.class, new AsnOctetStringOerSerializer())
        .register(AsnUint64Codec.class, new AsnOctetStringOerSerializer())
        .register(AsnUtf8StringCodec.class, new AsnCharStringOerSerializer())
        .register(AsnUtf8StringBasedObjectCodec.class, new AsnCharStringOerSerializer());

    return new CodecContext(mappings, serializers);
  }
```

Reply:
LGTM!